### PR TITLE
Feature/submit

### PR DESCRIPTION
**Pull request by magjo21**

### CODE REVIEW

**Comment by AstridOttmann** [comment]
It works! :-)
It would be nice to get back to the landing page after "see ya" (but maybe that's not part of your story).
I have noticed, that you use the same submit function on all pages. So you could outsource the function and use for all pages the same

File: pages/good-page.js [hunk 1]
```diff
@@ -15,6 +15,10 @@ export default function GoodPage() {
   const router = useRouter();
   const [buttons, setButtons] = useState(betterButtons);
 
+  function handleSubmit() {
+    window.localStorage.setItems(`buttons`, JSON.stringify(buttons));
```
Comment (doemser) [inline pages/good-page.js:19]: use another key, good-aspects for example

File: pages/middle-page.js [hunk 2]
```diff
@@ -15,6 +15,10 @@ export default function MiddlePage() {
   const router = useRouter();
   const [buttons, setButtons] = useState(betterButtons);
 
+  function handleSubmit() {
+    window.localStorage.setItems(`buttons`, JSON.stringify(buttons));
+  }
```
Comment (doemser) [inline pages/middle-page.js:20]: key

File: pages/thanks.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+export default function Goodbye() {
+  return (
+    <>
+      <h1>SEE YA</h1>
+    </>
```
Comment (doemser) [inline pages/thanks.js:5]:
```suggestion
      <h1>SEE YA</h1>
```

**Review by doemser — CHANGES_REQUESTED** [review]

**Pull request merged**